src/main/java/com/github/princesslana/eriscasper/data/util/Jackson.java

```diff
@@ -17,6 +19,9 @@ public static ObjectMapper newObjectMapper() {
     jackson.registerModule(new GuavaModule());
     jackson.registerModule(new Jdk8Module());
     jackson.registerModule(new JavaTimeModule());
+    SimpleModule module = new SimpleModule();
+    module.addDeserializer(AuditLogChange.class, new AuditLogChangeDeserializer());
```

Comment:
Do we need to do this? Is using a `@JsonDeserialize` annotation not sufficient?

Reply:
It's not sufficient, just due to the mixed values and such it won't deserialize correctly. It doesn't fit tests or match properly.

Reply:
I don't follow.

Wouldn't adding `@JsonDeserialize(with=AuditLogDeserializer.class)` use exactly the same code as this module sets up?

Reply:
Let me test it.

Reply:
Yeah looks like it does the same thing, never knew it had option for a deserializer.